sympy/printing/octave.py

```diff
@@ -392,6 +392,10 @@ def _print_lowergamma(self, expr):
                                               self._print(expr.args[0]))
 
 
+    def _print_sinc(self, expr):
+        return "sinc(%s / pi)" % self._print(expr.args[0])
```

Comment:
It needs to be parenthesized, for instance, so that `sinc(x + 1)` prints as `sinc((x + 1) / pi)`, not `sinc(x + 1 / pi)`.